app/javascript/listings/__tests__/ListingDashboard.test.jsx

```diff
@@ -180,6 +180,10 @@ const listings = {
   selectedListings: 'user',
 };
 
+global.window.timestampToLocalDateTimeLong = () =>
```

Comment:
It would be nice if there is a better way to mock the global util function (or use the actual util function from **app/assets/javascripts/utilities/localDateTime.js** in jest).

Reply:
cc @nickytonline - maybe he knows

Reply:
You can do what I did for the `timeAgo` function, https://github.com/forem/forem/blob/master/app/assets/javascripts/utilities/timeAgo.js. And here is the corresponding test, https://github.com/forem/forem/blob/master/app/javascript/utilities/__tests__/timeAgo.test.js.

Importing the file, even though nothing is exported from the file will still give you access to timeAgo, because it's set on the `globalThis` when the import runs.